tonic/src/request.rs

```diff
@@ -7,6 +8,23 @@ pub struct Request<T> {
     message: T,
 }
 
+/// A trait that is implemented for all RPC request request types.
+pub trait IntoRequest {
+    /// The RPC request type
+    type Message;
+
+    /// Wrap `Message` in a `tonic::Request`
+    fn into_request(self) -> Request<Self::Message>;
+}
+
+#[doc(hidden)]
```

Comment:
why is this one doc hidden and the other is not?

Reply:
Because IntoRequest is explicitly generated and there are blanket implementations for this one. Should I expose this one as well?

Reply:
Yeah, I think it would make it less confusing no?

Reply:
Yep. Done.